### PR TITLE
Add sidebar and change first letters size at some places

### PR DESCRIPTION
**Pull request by HOK405**
- Add sidebar
- Change first letters size at some words
- Use "Produce Sans" font at the whole application

### CODE REVIEW

File: client/src/app/shared/components/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<nav class="sidebar bg-light" *ngIf="isAuthenticated" [ngClass]="{'sidebar-collapsed': isCollapsed}">
+    <ul class="navbar-nav">
+      <li class="nav-item" *ngIf="isAuthenticated">
```
Comment (msyniuk-reenbit) [inline client/src/app/shared/components/sidebar/sidebar.component.html]:
```suggestion
      <li class="nav-item">
```

Not needed, also not needed below checks for `*ngIf="isAuthenticated"`

File: client/src/app/shared/components/sidebar/sidebar.component.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  private destroy$ = new Subject<void>();
+
+  constructor(private authService: AuthService, private router: Router) {}
+
+  ngOnInit(): void {
+    this.authService.authStatus$
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((status) => {
+        this.isAuthenticated = status;
+      });
```
Comment (msyniuk-reenbit) [inline client/src/app/shared/components/sidebar/sidebar.component.ts]:
```javascript
 this.authService.authStatus$
      .pipe(takeUntil(this.destroy$))
      .subscribe((status) => {
        this.isAuthenticated = status;
 });
``` 

You can shorten this operation by using `| async` pipe. So, for that you will need to make in
 `constructor(public authService: AuthService, private router: Router) {}`
 
`authService` as `public` to be able to use it in HTML, than in HTML you can write like this: 
 
 ```HTML
 <nav class="sidebar bg-light" *ngIf="authService.authStatus$ | async" [ngClass]="{'sidebar-collapsed': isCollapsed}">
 ```
 
 The benefits of this are: 
  1. `| async` pipe automatically subscribes AND **unsubscribes**  from your Observable, so you don't need to handle it's unsubscr. in `ngOnDestroy` method
  2. Clear and more readable code in HTML
  3. Less code in `.ts` files
 
 And you have to just remove the whole `ngOnInit`, `ngOnDestoy` method + implementation from the `class`, `destroy$` variable, as you don't need to handle unsubscription any more.

File: client/src/app/shared/components/sidebar/sidebar.component.html (hunk 1 above)
Comment (HOK405) [reply client/src/app/shared/components/sidebar/sidebar.component.html]: Fixed

File: client/src/app/shared/components/sidebar/sidebar.component.ts (hunk 2 above)
Comment (HOK405) [reply client/src/app/shared/components/sidebar/sidebar.component.ts]: Fixed

**Review by msyniuk-reenbit — APPROVED** [review]

**Pull request merged**